ui/modules/apps/BeamMP-Session/app.html

```diff
@@ -4,21 +4,22 @@
 		<div>
 			<button type="button" id="quit-button" name="button" class="buttons" ng-click="mpquit()">Quit</button>
 		</div>
-		<div class="block" id="server-name-block" style="display: none;">
-			<div class="marquee">
-				<span class="session-status"></span>
-				<span class="session-status"></span> 
-				<!-- duplicated for the marquee -->
+		<div class="info-container">
+			<div class="block" id="server-name-block" style="display: none;">
+				<div class="marquee">
+					<span class="session-status"></span>
+					<span class="session-status"></span> 
+					</div>
```

Comment:
Indentation

```suggestion
				</div>
```


Also inconsistent indentations in the style file class changes